scripts/test_catalyst.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# USAGE: test_catalyst.sh pod build_mode
+#
+# Builds and run tests for Catalyst since it's not yet supported by `pod lib lint`
+# The second argument should be "build" or "test". "test" indicates both build and test.
+
+pod="$1"
+build_mode="$2"
+
+set -xeuo pipefail
```

Comment:
Note: if you put this above the assignments this will catch `$1` and `$2` being unset for you.